<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDiv</class>
 <widget class="QWidget" name="AddDiv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сложение и деление</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(117, 64, 238);
border-radius: 20px;
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 20pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>Сложение/Деление</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 30px;
font: 14pt &quot;Inter&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(230, 230, 230);
}


</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 0px 0px 5px 5px;
font: 11pt &quot;Inter&quot;;

color: rgb(255, 255, 255);
}




</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 30px;
font: 14pt &quot;Inter&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(230, 230, 230);
}


</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secNum_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 30px;
font: 14pt &quot;Inter&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(230, 230, 230);
}


</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="firstNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="secNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="resLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>750</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 10px;
font: 10pt &quot;Inter&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}


QPushButton:hover {
	background-color: rgb(45, 45, 45);
}



</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>750</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 10px;
font: 10pt &quot;Inter&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}


QPushButton:hover {
	background-color: rgb(45, 45, 45);
}



</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
font: 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
font: 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Второе число:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
font: 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Первое число:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
